hedera-node/hedera-app/src/main/java/com/hedera/node/app/services/ServiceScopeLookup.java

```diff
@@ -103,6 +103,7 @@ public String getServiceName(@NonNull final TransactionBody txBody) {
                     TOKEN_PAUSE,
                     TOKEN_UNPAUSE,
                     TOKEN_UPDATE_NFTS,
+                    TOKEN_AIRDROP,
```

Comment:
This should have `TOKEN_CANCEL_AIRDROP` also, since this PR is adding TokenCancelAirdrop also